wsf_scraping/pipelines.py

```diff
@@ -2,17 +2,32 @@
 import os
 import logging
 from tools.dbTools import insert_article
+from tools.cleaners import parse_keywords_files
 from scrapy.utils.project import get_project_settings
 from pdf_parser.pdf_parse import (get_pdf_document, parse_pdf_document,
                                   grab_section)
 
 
 class WsfScrapingPipeline(object):
+    def __init__(self):
+        """Initialise the pipeline, giveng it the settings and keywords."""
+
+        self.settings = get_project_settings()
```

Comment:
another common approach is to have the consts in a file and the just import them in the beggining 

`from settings import *`